docs/scos/user/intro-to-spryker/contribute-to-the-documentation/edit-the-documentation-locally.md

```diff
@@ -0,0 +1,71 @@
+
+
+## Find the documents you want to edit on your machine
+
+At this point, you are most likely to know which document you want to edit. To find the document on your machine, you need to look up its path in the URL. Open the document on our website and check the path starting from `/docs/`. For example, the URL of the document you are reading is `https://docs.spryker.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/edit-the-documentation-locally.html`. The path of this document on your machine is `docs/scos/user/intro-to-spryker/contribute-to-the-documentation/edit-the-documentation-locally.md`. The path is relative to where you chose to store the repository.
+
+## Edit documents
+
+1. Open the document you want to edit in an editor.
+2. Add the needed changes.
+    You'd help us a lot by following [Markdown syntax](/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/markdown-syntax.html) and [general rules](https://docs.spryker.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/style-formatting-general-rules.html), but it's completely optional. We are grateful for your contribution in any form.
+3. Save the edited document.
+4. In GitHub Desktop, double-check your changes in the **Changes** tab.
+5. Optional: Below the list of edit files, enter the name and description of your changes.
+6. To add your changes to the branch, click **Commit to {BRANCH_NAME}**.
+7. To publish the changes on GitHub, click **Push origin**.
+    If you are editing in several sessions, it is good practice to commit and push after each session. If something happens to the changes on your machine, they will be safe on GitHub, and you will be able to keep working on them.
+
+![Comit and push](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/edit-the-documentation-locally.md/Commit+and+push.mp4)    
+
+## Submit changes for publishing
+
+1. To prepare the changes to be published on the documentation website, click **Create Pull Request**.
+    This opens the **Open a pull request** page with prefilled name and description.
+2. Optional: Update the name to describe your changes.     
+3. Add a description of what you've changed.
+4. Click **Create pull request**.
+    This refreshes the page and displays the created pull request. The status is **Open**.
+
+![Create a pull request](https://spryker.s3.eu-central-1.amazonaws.com/docs/scos/user/intro-to-spryker/contribute-to-the-documentation/edit-the-documentation-locally.md/Create+a+pull+request.mp4)    
+
+That's it. We'll take it from here. During review, we may post comments for your changes. Notifications about comments and PR status changes will be emailed to the email address of your GitHub account. After the pull request is merged, the changes will appear on the website.     
```

Comment:
suggestion to avoid tautology
```suggestion
That's it. We'll take it from here. During the review, we may post comments for your changes. Notifications about comments and PR status changes will be sent to the email address of your GitHub account. After the pull request is merged, the changes will appear on the website.     
```